lib/BIGSdb/Plugins/PhyloViz.pm

```diff
@@ -0,0 +1,415 @@
+#PhyloViz.pm - phylogenetic inference and data visualization for sequence based typing methods for BIGSdb
+#Written by Emmanuel Quevillon
+#Copyright (c) 2016, Institut Pasteur, Paris
+#E-mail: tuco@pasteur.fr
+#
+#This file is part of Bacterial Isolate Genome Sequence Database (BIGSdb).
+#
+#BIGSdb is free software: you can redistribute it and/or modify
+#it under the terms of the GNU General Public License as published by
+#the Free Software Foundation, either version 3 of the License, or
+#(at your option) any later version.
+#
+#BIGSdb is distributed in the hope that it will be useful,
+#but WITHOUT ANY WARRANTY; without even the implied warranty of
+#MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#GNU General Public License for more details.
+#
+#You should have received a copy of the GNU General Public License
+#along with BIGSdb.  If not, see <http://www.gnu.org/licenses/>.
+package BIGSdb::Plugins::PhyloViz;
+use strict;
+use warnings;
+use 5.010;
+use parent qw(BIGSdb::Plugin);
+use List::MoreUtils qw(uniq);
+use Log::Log4perl qw(get_logger);
+my $logger = get_logger('BIGSdb.Plugins');
+use Apache2::Connection ();
+use constant DEBUG => 0;
+
+sub get_attributes {
+	my ($self) = @_;
+	my %att = (
+		name        => 'PhyloViz',
+		author      => 'Emmanuel Quevillon',
+		affiliation => 'Institut Pasteur, Paris',
+		email       => 'tuco@pasteur.fr',
+		description => 'Creates phylogenetic inference and data visualization for sequence based typing methods',
+	        category    => 'Analysis',
+		buttontext  => 'PhyloViz',
+	        # Set menutext to empty string to not display it on index page (See BIGSdb::IndexPage line 329)
+	        menutext    => '',
```

Comment:
That's what I noticed. I've made a modifcation in IndexPage.pm to support this undefined 'menutext' attribute.
But as you said below, removing 'application' from 'section' will solve all the problems